lib/components/Editor/CustomExtensions/KeyboardShortcuts/ExtensionConfig.js

```diff
@@ -1,18 +1,25 @@
 import { Extension } from "@tiptap/core";
+import { isNilOrEmpty } from "utils/common";
+import { generateKeyboardShortcuts } from "./helpers";
 
-const KeyboardShortcuts = ({ handleSubmit }) =>
-  Extension.create({
+const KeyboardShortcuts = ({ handleSubmit, shortcuts }) => {
+  if (isNilOrEmpty(shortcuts)) return;
```

Comment:
if we do this here, won't this affect the handleSubmit? the user may have passed a onSubmit prop which is invoked with shortcut `Mod-Enter` by default **even when `shortcuts` is not specified**